packages/core/src/common/cluster/request-namespace-list-permissions.injectable.ts

```diff
@@ -47,23 +47,17 @@ const requestNamespaceListPermissionsForInjectable = getInjectable({
           const { resourceRules } = status;
 
           return (resource) => {
-            const resourceRule = resourceRules.find(({
-              apiGroups = [],
-              resources = [],
+            const rules = resourceRules.filter(({
```

Comment:
Did I get this right?
- defaults are now `["*"]`
- instead of checking verbs from the first match we check them from all matches

Reply:
Yes that is correct.

I think that the section·second point is more important but I haven't been able to test this.